Docs/Product/AuthNZ-PRDs_POST_V0_1_TRACKER.md

```
@@ -0,0 +1,170 @@
+  - `python Helper_Scripts/rg_stage9a_parity_window.py exercise --api-key "$SINGLE_USER_API_KEY" --timeout 10`
+- Traffic smoke checklist (prints HTTP status codes):
+  - `curl -sS -o /dev/null -w "mcp.status %{http_code}\\n" "$BASE/api/v1/mcp/status"`
+  - `curl -sS -o /dev/null -w "audio.health %{http_code}\\n" "$BASE/api/v1/audio/transcriptions/health"`
+  - `curl -sS -o /dev/null -w "chat.commands %{http_code}\\n" -H "X-API-KEY: $SINGLE_USER_API_KEY" "$BASE/api/v1/chat/commands"`
+  - `curl -sS -o /dev/null -w "emb.providers %{http_code}\\n" -H "X-API-KEY: $SINGLE_USER_API_KEY" "$BASE/api/v1/embeddings/providers-config"`
+  - `curl -sS -o /dev/null -w "evals.rate_limits %{http_code}\\n" -H "X-API-KEY: $SINGLE_USER_API_KEY" "$BASE/api/v1/evaluations/rate-limits"`
+  - `curl -sS -o /dev/null -w "workflows.auth_check %{http_code}\\n" -H "X-API-KEY: $SINGLE_USER_API_KEY" "$BASE/api/v1/workflows/auth/check"`
+  - `curl -sS -o /dev/null -w "rag.health %{http_code}\\n" "$BASE/api/v1/rag/health"`
+  - `curl -sS -o /dev/null -w "auth.login(bad) %{http_code}\\n" -X POST "$BASE/api/v1/auth/login" -H "Content-Type: application/x-www-form-urlencoded" --data "username=bad&password=bad"`
+- Character chat create (emits `character_chat.default` decisions when `RG_ENABLED=1`; legacy limiter is disabled by default in single-user mode):
+    - `curl -sS -o /dev/null -w "chats.create %{http_code}\\n" -H "X-API-KEY: $SINGLE_USER_API_KEY" -H "Content-Type: application/json" -X POST "$BASE/api/v1/chats/" -d '{"character_id":999999,"title":"stage9a"}'`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)